pkg/cmd/cli.go

```diff
@@ -2,18 +2,28 @@ package cmd
 
 import (
 	"glide/pkg"
+	"glide/pkg/config"
 
 	"github.com/spf13/cobra"
 )
 
+var cfgFile string
+
 // NewCLI Create a Glide CLI
 func NewCLI() *cobra.Command {
 	// TODO: Chances are we could use the build in flags module in this is all we need from CLI
 	cli := &cobra.Command{
-		Use:     "",
-		Version: pkg.GetVersion(),
+		Use:     "glide",
+		Short:   "🐦Glide is an open source lightweight high-performant model gateway",
```

Comment:
Spelling: "🐦Glide is an open-source, lightweight, high-performance model gateway."

Reply:
Applied in https://github.com/modelgateway/glide/pull/37/commits/85afa75e9b6b8225b21e1aa0d3161734a2996c1b ✅